### ISSUE TITLE
export reports: ly:grob-properties variable not found in lilypond 2.24.0

### ISSUE THREAD

**Issue by adsieben** [openlilylib/lilypond-export#29]
It looks like that the ly:grob-properties function has been removed:
https://sourceforge.net/p/testlilyissues/issues/5889/
Therefore the export won't work due to the use of it in lilypond-export/api.scm.

Best regards/Liebe Grüße
Andre

**Comment by reckel-jm** [comment]
Yes, it should be replaced with `ly:grob-property` - anyone knows how to implement this?

**Comment by kair0span** [comment]
Some solution for the problem? I run into the following issue: Starting lilypond 2.24.4 [bach.ly]...
Processing `/var/folders/13/tf6n6x2x2nq6jz4tg18fg5bh0000gn/T/frescobaldi-jqmfhms9/tmpp579_tyb/bach.ly'
Parsing...

oll-core: library infrastructure successfully loaded.
[ice-9/format.scm:52:8](https://github.com/openlilylib/lilypond-export/issues/0): In procedure format:
format: bad destination `~A.~A'
Exited with return code 1.
